docs/documentation/intro.md

```diff
@@ -1,12 +1,67 @@
-# NATS.Net
+# Introduction
 
-NATS.Net is a .NET client for the Open Source [Connective Technology for Adaptive Edge & Distributed Systems - NATS](https://nats.io/)!
+NATS.Net is a .NET client for the open source [Connective Technology for Adaptive Edge & Distributed Systems - NATS](https://nats.io/)!
 It's build on top of the modern .NET platform, taking advantage of all the high performance features and
 asynchronous programming model.
 
-NATS.Net, just like NATS, is Open Source as is this documentation.
-Please [let us know](https://natsio.slack.com/channels/dotnet) if you have updates and/or suggestions for
-these docs. You can also create a Pull Request using the Edit on GitHub link on each page.
+NATS.Net, just like NATS, is open source as is this documentation.
+Please [let us know](https://natsio.slack.com/channels/dotnet) if you have updates or suggestions for
+these docs. You can also create a Pull Request in GitHub using the _Edit this page_ link on each page.
+
+## Quick Start
+
+You can [download the latest](https://nats.io/download/) `nats-server` for your platform and run it without any arguments.
```

Comment:
I would also mention docker image for nats-server as an alternative.

Reply:
added